src/lib/UserPositionTracker.js

```diff
@@ -0,0 +1,54 @@
+import { shouldLocate, addListener } from './savedState';
+
+/**
+ * The UserPositionTracker allows tracking changes to the user position.
+ * It respects the global setting for tracking on the map, and only
+ * collects positions while it is turned on.
+ */
+export default class UserPositionTracker {
+  userLocation: null | Position = null;
```

Comment:
+0: We could use the optional type here, righ? `userLocation: ?Position = null;`

Reply:
Yes, would work - optional allows `undefined` & `null`, I did it like this to indicate the null possibility clearer.

Reply:
Done.